tutorpod_autoscaling/templates/pod-autoscaling/k8s/keda.yml

```diff
@@ -0,0 +1,23 @@
+{% for worker, autoscaling in POD_AUTOSCALING_KEDA_CONFIG.items() %}
+---
+apiVersion: keda.sh/v1alpha1
+kind: Keda
```

Comment:
The object kind is `Keda` of [ScaledObject](https://keda.sh/docs/2.15/reference/scaledobject-spec/) ?